homeassistant/components/homekit/util.py

```diff
@@ -448,13 +448,13 @@ def density_to_air_quality_nitrogen_dioxide(density: float) -> int:
 
 def density_to_air_quality_voc(density: float) -> int:
     """Map VOCs µg/m3 to HomeKit AirQuality level."""
```

Comment:
My main grind with this PR is: We are breaking it again; and to be honest, that is a bit unacceptable to break twice in such a short time.

I'm not using this myself, so have no personal opinion on it, but I really do want an extensive reasoning and definitive decision documented in the code comments here, that will block/prevent any breaking change to this again in the future.

Reply:
Thank you for reviewing my pull request! I've added code comments to indicate the source of the values used.

Reply:
The VOC mappings in the original PR that added them https://github.com/home-assistant/core/pull/81217 don't seem to align well with any standard AFAICT.

Looking back at this, it looks like there were no previous breaking changes for VOC, only [PM2.5](https://github.com/home-assistant/core/pull/76358) and [PM1.0](https://github.com/home-assistant/core/pull/81217)

Reply:
I just wanted to be extra careful with the pull request regarding breaking changes.
But it's great to hear that there were no previous breaking changes!

Is there something else I can do to help move this forward?

Reply:
Your PR makes sense. We are good to go now.